packages/hoppscotch-common/src/newstore/collections.ts

```diff
@@ -61,7 +61,7 @@ export function navigateToFolderWithIndexPath(
 
   let target = collections[indexPaths.shift() as number]
 
-  while (indexPaths.length > 0)
+  while (indexPaths.length > 0 && target)
```

Comment:
A safeguard was added here since the below exception was observed while importing collections into the personal workspace and expanding the collection tree.

![image](https://github.com/hoppscotch/hoppscotch/assets/25279263/b0756056-ff8f-4aeb-8292-9fec5c3351e7)